samples/blender/07_amass.py

```diff
@@ -0,0 +1,59 @@
+"""
+>>> python -m samples.blender.07_amass
+
+This is a script to demonstrate importing Amass motion and applying it to SMPL-XL model.
+Before running this script, please download SMPL-XL model and Amass dataset first,
```

Comment:
For users not knowing SMPL-XL, it's better to offer a ref link to the paper here.

Reply:
Fair suggestion. Add paper links in [`b54c643` (#12)](https://github.com/openxrlab/xrfeitoria/pull/12/commits/b54c643570638bb0ed3d5b31cdbfaad1ba086b45#diff-9be48bfb056bce8b28b26d9d062d75efd32a4312065b77ec113dda52d7700753).